### DISCUSSION TITLE
How to contribute to the wiki?

### DISCUSSION THREAD

**Discussion by ulisses-cruz** [Julien-cpsn/ATAC#92] [Q&A] [answered]
I would like to contribute in documentation to the Wiki. 
How can I do that?

**Comment by ccoVeille** [comment]
It's a pain due to GitHub

Please read this

https://gist.github.com/omaraboumrad/35654da0a376c57a2e0ab4d92ad0c339

**Reply by ulisses-cruz** [reply]
I did find that link before. But it did not work for me. After forking the repo I went to the wiki but I never saw the error mentioned in the gist when trying to create a page, or the wiki pages from the  original repo. Thanks though.

**Comment by Julien-cpsn** [comment, marked as answer]
@ulisses-cruz @ccoVeille Hello guys!
The wiki will soon be deleted. We are now using my own website, which uses [Docusaurus](https://docusaurus.io/). It is much more practical to read and edit.
Here's the URL: https://atac.julien-cpsn.com/
(click the edit button on the page you want to edit, or just create a PR on the [repo](https://github.com/Julien-cpsn/ATAC-website))

Have a great day

**Reply by ccoVeille** [reply]
That's a good news because GitHub Wiki is a shame 😅

**Reply by ulisses-cruz** [reply]
Thanks. That is great. Looks like the docs I wanted to contribute are already well explained in the Docusaurus docs. Great work!

**Reply by Julien-cpsn** [reply]
> Thanks. That is great. Looks like the docs I wanted to contribute are already well explained in the Docusaurus docs. Great work!

A lot of pages are still missing, do not hesitate to contribute!
